pkg/products/grafana/reconciler.go

```diff
@@ -79,43 +79,42 @@ func NewReconciler(configManager config.ConfigReadWriter, installation *integrea
 		}
 	}
 
-	logger := logrus.NewEntry(logrus.StandardLogger())
 	return &Reconciler{
 		ConfigManager: configManager,
 		Config:        config,
 		installation:  installation,
 		mpm:           mpm,
-		logger:        logger,
+		log:           logger,
 		Reconciler:    resources.NewReconciler(mpm),
 		recorder:      recorder,
 	}, nil
 }
 
 func (r *Reconciler) Reconcile(ctx context.Context, installation *integreatlyv1alpha1.RHMI, product *integreatlyv1alpha1.RHMIProductStatus, client k8sclient.Client) (integreatlyv1alpha1.StatusPhase, error) {
-	logrus.Infof("Start Grafana reconcile")
+	r.log.Info("Start Grafana reconcile")
```

Comment:
I like the idea of having "START" and "FINISH" of reconciling highlighted, wonder if you think we could do the same for other products? It seems there's a bit of inconsistency, for example, fuse is missing the "start" of the reconciling log. To me, logs are much easier to read when the information they provide is consistent and I think either removing the above the line or having it done the same way for other products would be great!

Reply:
We don't yet have standards around this. The main purpose of this JIRA was to get the contextual and structure logging in place. It wasn't to fix all logging issues or set the standards. It's a good suggestion but I don't think it should be in scope of this JIRA.

Reply:
Thanks Brian, kinda felt that way :D sorry for pointing the actual contents of the logs in your PR then!